pawan/html/practice_assignmenr/Programming_Exercises4.html

```diff
@@ -0,0 +1,61 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <title>Resume</title>
+        <style> 
+        table , th , td {
+        border: 2px solid black;}
+        </style>
+    </head>
+    <body>
+        <h1>Pawan Sharad Jagtap</h1>
+            <p><strong>pavanj10may@gmail.com</strong></p>
```

Comment:
these are all siblings, so no need of tab spaces...
Did you know the child-parent-sibling relationship in HTML ?